content/writing/digital-approaches-for-social-change-organisations.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "Digital approaches for social change organisations"
+date: 2021-09-13
+intro: Learnings from our collaboration with the Joseph Rowntree Foundation
+---
+
+## Context
+
+What does a social change organisation look like online? We recently worked with the [Joseph Rowntree Foundation](https://www.jrf.org.uk/) to explore this question through research and rapid prototyping.
```

Comment:
Perhaps we could have an image at the start. The first bite with the eye and that?

Looks more appealing https://deploy-preview-74--commonknowledge.netlify.app/writing/digital-approaches-for-social-change-organisations